ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -1423,6 +1460,7 @@ static PrjFS_Result RecursivelyMarkAllChildrenAsInRoot(const char* fullDirectory
                 CombinePaths(fullDirectoryPath, relativePath, fullPath);
                 if (!SetBitInFileFlags(fullPath, FileFlags_IsInVirtualizationRoot, true))
                 {
+                    LogWarning("RecursivelyMarkAllChildrenAsInRoot: FileFlags_IsInVirtualizationRoot not set for fullPath=%s", fullPath);
```

Comment:
Minor: 

"FileFlags_IsInVirtualizationRoot not set for " could be interpreted as meaning that `FileFlags_IsInVirtualizationRoot` was not set when the code expected it to be set.

Maybe change this to something like : "SetBitInFileFlags failed to set FileFlags_IsInVirtualizationRoot"?